manager/integration/framework/strategy/__init__.py

```diff
@@ -0,0 +1 @@
+from .strategy import Strategy
```

Comment:
I feel this package is not really necessary. You probably can just have strategy.py in the root package folder, but you can refactor anytime in the future. KISS first.